src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_scenarios_non_ascii_chars.py

```diff
@@ -0,0 +1,64 @@
+# This Python file uses the following encoding: utf-8
```

Comment:
Can you move this to under the license?

Reply:
Nope, this needs to go above the license in either the first or second line.

Reply:
Okay I see from https://www.python.org/dev/peps/pep-0263/